packages/cli/test/test-utils.js

```diff
@@ -15,3 +19,22 @@ exports.testSetUpGen = function(genName, arg) {
   env.register(genName, 'loopback4:' + name);
   return env.create('loopback4:' + name, arg);
 };
+
+/**
+ * Setup the target directory with the required package.json and folder
+ * structure.
+ * @param {string} tmpDir Path to the temporary directory to setup
+ */
+exports.setupTmpDir = function(tmpDir) {
```

Comment:
This function should be from `@loopback/workspace` in the future to create a dummy project.

Reply:
Please use a more descriptive name. `setupTmpDir` does not tell us anything about what is being set up in the target directory. I am proposing to use `givenAnApplicationDir` or `setupAProjectDir` or similar.

Reply:
+1 for `givenAnApplicationDir`